servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -209,52 +244,37 @@ def _process_components(self, instance_info):
         collected_components = self._batch_collect(self._batch_collect_components, instance_info)
 
         for component in collected_components:
+            try:
+                config_item = ConfigurationItem(component, strict=False)
+                config_item.validate()
+            except DataError as e:
+                self.log.warning("Error while processing properties - {}".format(e))
```

Comment:
Add the component `sys_id` and `name` to the log

Reply:
added.